lms/urls.py

```diff
@@ -61,6 +61,9 @@
 
     url(r'^heartbeat$', include('heartbeat.urls')),
 
+    # TODO jimabramson: put this in the right place
+    url(r'^api/profile_images/', include('openedx.core.djangoapps.profile_images.urls')),
```

Comment:
I agree-- either it should be a general file upload API, or it should be within the user_api app.